services/movie_session.py

```diff
@@ -20,15 +19,17 @@
     return queryset
 
 
-def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
-    return MovieSession.objects.get(id=movie_session_id)
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = Ticket.objects.filter(
+        movie_session_id=movie_session_id).values("row", "seat")
+    return list(tickets)
 
 
 def update_movie_session(
     session_id: int,
     show_time: str = None,
```

Comment:
Consider using `datetime` objects for `show_time` instead of `str` for better type safety and to leverage Django's date/time handling capabilities.